_delphi_utils_python/tests/test_geomap.py

```diff
@@ -310,3 +311,24 @@ def test_get_geos(self):
         assert len(gmpr.get_geo_values("fips")) == 3287
         assert len(gmpr.get_geo_values("state_id")) == 60
         assert len(gmpr.get_geo_values("zip")) == 32976
+
+    def test_megafips_zeroed(self):
+        gmpr = GeoMapper()
+        pd.testing.assert_frame_equal(
+            gmpr.replace_geocode(self.fips_data_2, "fips", "state_id", pop_col="population"),
```

Comment:
could you use `state_code` here instead, it makes it easier to see what happened to `04000`

Reply:
good call, will do